tests/e2e/apps/hop/app.go

```diff
@@ -34,13 +34,39 @@ import (
 	"google.golang.org/grpc"
 
 	"istio.io/istio/tests/e2e/apps/hop/config"
+	"istio.io/istio/tests/e2e/framework"
+	"istio.io/istio/tests/e2e/util"
 )
 
 var (
-	timeout = flag.Duration("timeout", 15*time.Second, "Request timeout")
-	version = flag.String("version", "", "Server version")
+	timeout     = flag.Duration("timeout", 15*time.Second, "Request timeout")
+	version     = flag.String("version", "", "Server version")
+	hopYamlTmpl = "tests/e2e/framework/testdata/hop.yam.tmpl"
```

Comment:
The hopTemplate struct defined here points to this file, so if people want to create their own template they would need to  create their own struct as well.